web/themes/custom/usagov/templates/menu--sidebar_first.html.twig

```diff
@@ -15,7 +15,22 @@ A short summary of the behavior of the sidebar nav:
 
 {% import _self as menus %}
 
-{{ menus.explore(items, 0, null, null) }}
+{# {% set relatedPages = drupal_entity('node', items.url.routeParameters.node )['#node'].field_related_pages %} #}
+
+
+{% if path('<current>') matches '#\/agencies\/*#'%}    
+  {% include 'menu--agency-sidebar.html.twig' %}
+
+{% elseif path('<current>') matches '#\/states\/*#' %}    
+  {% include 'menu--agency-sidebar.html.twig' %}
+
+{% elseif path('<current>') matches '#\/es\/agencias\/*#' or path('<current>') matches '#\/es\/estados\/*#'%}    
```

Comment:
While you're making updates, why not combine lines 21 and 24 the way you did on line 27 here?

Reply:
Oops! I just realized that we actually want to take out the clauses for '#\/agencies\/*#' and '#\/states\/*#' -- /agencies isn't actually a page, and /states should have the dynamically-generated menu (which would be why I'm seeing "State Governments" twice on /states now).